### ISSUE TITLE
Permission Granting panel does not show up

### ISSUE THREAD

**Issue by qbalin** [AzureAD/azure-activedirectory-library-for-nodejs#60]
Hi, I used [this sample of code][1] for test purposes: I manage to authenticate an account and get back the accessToken, but at no time am I asked to grant a set of permissions, yet my application registered on AzureAd requires a bunch of them.

I even tried to add prompt=consent in the request URL, but then I get an "incorrect request" message...

What am I doing wrong? 

[1]: https://github.com/AzureAD/azure-activedirectory-library-for-nodejs/blob/master/sample/website-sample.js

**Comment by RandalliLama** [comment]
@qbalin sorry for the ridiculously late response.  Did you figure this out?

I would only expect consent dialogues if you have a multi-tenant app and the user is signing in from a different tenant than the app is registered in.

**Comment by qbalin** [comment]
Hi! 
Alas, I did so many manipulations I could not help someone running into the same problem.

When I started working on this, the OAuth flow was very new to me, I did not get that the permissions were only asked to deliver the request token, but not to get or refresh the access token. Also, I mainly got confused with the fact that Microsoft delivers different token for several APIs (office365, Graph...) through the same endpoint.

I ended up using this library, as I happen to work with Meteor:https://github.com/djluck/azure-active-directory, and I made a commit allowing the request of tokens of any kind (office 365, Graph, ...). 

Thanks again, and sorry for being so blurry, I really forgot how I got entangled, and then disentangled.
